### DISCUSSION TITLE
OFF Topic - liballeg.org apparently defaced

### DISCUSSION THREAD

**Discussion by robertspereira** [liballeg/allegro5#1489] [General]
Hello maintainer,

I just tried to access the URL liballeg.org and the main page is displaying a type of hate page.

Please check this asap. Other pages are unavailable (404 not found code).

**Comment by robertspereira** [comment]
It is in fact being redirected to another URL.

**Reply by miquelfire** [reply]
I saw this being talked about in Discord. They are aware of this.

**Reply by robertspereira** [reply]
Thanks for the reply. I hope they solve it quickly.

**Reply by miquelfire** [reply]
Mark Oates made a post on Discord with a summary of things so far:

>As a lot of us are now aware, the domain liballeg.org is in an awkward state. My understanding of the situation is that Trent Gamblin is the current holder of the domain and has decided to use it for his own purposes with a domain redirection.
>
>Since this is obviously not what it's  intended for, and goes against what everybody put their hard work into, I'm creating this thread to document what has been done to contact Trent and remedy the situation.
>
>So, what I have gathered so far:
>
>Trent is not available on discord (has deleted his account)
>I reached out just a while ago to an email that I have for him (__@gamblin.ca)
>I sent a message on a.cc just a few minutes ago as well
>
>If everybody else could document their attempts to contact him here (if you feel comfortable doing so). Or, share any updates or information on the situation, that'd be great.
>
>We should continue to try to restore the domain for all the SEO juice it has. (In the mean time, manual and website is still accessible at liballeg.github.io)

**Reply by miquelfire** [reply]
They got control back of the old domain, so everything is back to normal.

**Reply by iopiiopi** [reply]
what's the discord link?  and also, i'm just curious, what was the hate page all about?  lol

**Reply by allefant** [reply]
Discord is here: https://discord.gg/f3Cd4TZzpp

It was a former disgruntled main developer. I don't remember what exactly the text was but it was quite unacceptable, bordering illegal.

**Reply by iopiiopi** [reply]
oh man, that sucks.  :(

thanks for the link though. :)
